### DISCUSSION TITLE
Whats the best way of checking if the client is connected?

### DISCUSSION THREAD

**Discussion by XcomReborn** [Bukk94/KickLib#5] [Q&A] [answered]
Is there an IsConnected bool somewhere ? I am creating a WPF form application and I would like to set the button text to Connect or Disconnect based on whether the client is connected or not. Whats the best way of doing this?

**Comment by Bukk94** [comment, marked as answer]
Hi, 
there was no `IsConnected` property available in the Client. 

But I just released a new version 1.8.0 where I have added this property so you can use it in your WPF app. I also updated .NET version of the KickLib to supported .NET 8. New package should be already available via NuGet.

**Reply by XcomReborn** [reply]
Thanks very much, thats great.
